tests/end-to-end/external-solution-integration/pkg/testsuite/create_knative_trigger.go

```diff
@@ -0,0 +1,77 @@
+package testsuite
+
+import (
+	"fmt"
+	"github.com/avast/retry-go"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/step"
+	"github.com/pkg/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1alpha12 "knative.dev/eventing/pkg/apis/eventing/v1alpha1"
```

Comment:
where is the `12` coming from in `v1alpha12` 🤔`v1alpha1` and then the IDE added a `2` at the end because of duplicate import names?